source/_templates/installations/elastic/common/configure_filebeat.rst

```diff
@@ -1,10 +1,28 @@
 .. Copyright (C) 2020 Wazuh, Inc.
 
-.. code-block:: yaml
+.. tabs::
 
-  output.elasticsearch.hosts: <elasticsearch_ip>:9200
-  output.elasticsearch.password: <elasticsearch_password>
 
-Replace ``elasticsearch_ip`` with the IP address or the hostname of the Elasticsearch server and ``elasticsearch_password`` with password previously generated for the ``elastic`` user.
+  .. group-tab:: Elasticsearch single-node cluster
```

Comment:
The tabs look great. 
Could we have "To learn more..." out of the tabs? I think that could be better. 

In addition to this, are we sure that `[` and `"` are necesaries? I would like to be sure, because I prefer this: 
`      output.elasticsearch.hosts: <elasticsearch_ip_node_1>:9200, <elasticsearch_ip_node_2>:9200, <elasticsearch_ip_node_3>:9200`
rather than: 
`      output.elasticsearch.hosts: ["<elasticsearch_ip_node_1>:9200", "<elasticsearch_ip_node_2>:9200", "<elasticsearch_ip_node_3>:9200"]`
The less characters, the lesser user error posibilities.